src/richie-front/scss/_colors.scss

```diff
@@ -0,0 +1,22 @@
+///
+/// Color palette as named variable
+///
+
+$black: #000000;
+$white: #ffffff;
+$gray7: #111111;
+$gray20: #333333;
+$gray40: #666666;
+$gray80: #cdcdcd;
+$gray97: #f7f7f7;
+$coral2: #f06d54;
+$cyan4: #009fa1;
+$deeppink4: #9c005d;
+$dodgerblue1: #08223c;
+$dodgerblue3: #335280;
+$dodgerblue5: #007aa6;
+$dodgerblue7: #168dc3;
+$firebrick3: #c7162b;
+$firebrick6: #f72c30;
```

Comment:
If these are color variables then maybe call them by a name that does not relate to colors? Maybe relate to role of the color? first/second? primary? main? ...

Reply:
I'd even say: reuse the names from bootstrap & override bootstrap's defaults?